tests/CI/install_server.sh

```diff
@@ -89,6 +89,15 @@ then
   # We ignore the Configuration for now because it is a bit special (the master needs to run in a separate process)
   # system_component is a space separated System (without the word System), and Component, without the 'Handler.py'.
   # For example "DataManagement TornadoFileCatalog"
+
+  if [[ "${USE_PYTHON3:-}" == "Yes" ]]; then
+    find "/home/dirac/LocalRepo/ALTERNATIVE_MODULES/DIRAC" -name 'Tornado*Handler.py' | grep -v Configuration | sed -e 's/Handler.py//g' -e 's/System//g'| awk -F '/' '{print $(NF-2), $NF}' > tornadoServices
+    cfgProdFile="${SERVERINSTALLDIR}"/diracos/etc/Production.cfg
+  else
+    find "${SERVERINSTALLDIR}"/DIRAC/ -name 'Tornado*Handler.py' | grep -v Configuration | sed -e 's/Handler.py//g' -e 's/System//g'| awk -F '/' '{print $(NF-2), $NF}' > tornadoServices
```

Comment:
I don't think that this is necessary as `tornadoservice` is not used, and I believe the `Production.cfg` files are symlinked

Reply:
where is the `system_component` (2 lines below) created? I have the impressione there's some merge or rebase issue.